charms/knative-operator/src/manifests/auth_manifests.yaml.j2

```diff
@@ -11,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Source: knative/operator/config/rbac/clusterrole_aggregated.yaml
```

Comment:
Let's keep this comment as a reference for the future

Reply:
The comment has been kept, but moved to [to the top](https://github.com/canonical/knative-operators/pull/196/files#diff-c4921c7c33bf4dffd77abf51591560f4cfe9e1844f8f87375adbaf9156cc3c49R1) to be consistent with the other .yaml.j2 files.